contrib/vagrant/provision-util.sh

```diff
@@ -154,6 +156,12 @@ os::provision::set-os-env() {
   ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
 }
 
+os::provision::get-bin-output-path() {
```

Comment:
There exists `os::build::find-binary` -- will this suit your needs instead of re-setting the `$PATH`? I'm not sure which we should be preferring but the `find-binary` calls are how we deal with the issue now.